docs/spec/openapi.yaml

```diff
@@ -0,0 +1,13317 @@
+                url: >-
+                  https://www.dropbox.com/s/2sn712vy1ovegw8/Prime_Numbers.txt?dl=0
+                settings:
+                  requested_visibility: public
+                  audience: public
+                  access: viewer
+                remove_expiration: false
+      parameters:
+        - name: Content-Type
+          in: header
+          schema:
+            type: string
+          example: application/json
+        - name: Dropbox-API-Path-Root
+          in: header
+          schema:
+            type: string
+          example: '{".tag": "namespace_id", "namespace_id": "2"}'
+        - name: Dropbox-API-Select-User
+          in: header
+          schema:
+            type: string
+          example: dbmid:FDFSVF-DFSDF
+      responses:
+        '200':
+          description: OK
+          headers:
+            X-Dropbox-Request-Id:
+              schema:
+                type: integer
+                example: '1234'
+            Content-Type:
+              schema:
+                type: string
+                example: application/json
+          content:
+            application/json:
+              schema:
+                type: object
+              example:
+                .tag: file
+                url: >-
+                  https://www.dropbox.com/s/2sn712vy1ovegw8/Prime_Numbers.txt?dl=0
+                name: Prime_Numbers.txt
+                link_permissions:
+                  can_revoke: false
+                  resolved_visibility:
+                    .tag: public
+                  revoke_failure_reason:
+                    .tag: owner_only
+                client_modified: '2015-05-12T15:50:38Z'
+                server_modified: '2015-05-12T15:50:38Z'
+                rev: a1c10ce0dd78
+                size: 7212
+                id: id:a4ayc_80_OEAAAAAAAAAXw
+                path_lower: /homework/math/prime_numbers.txt
+                team_member_info:
+                  team_info:
+                    id: dbtid:AAFdgehTzw7WlXhZJsbGCLePe8RvQGYDr-I
+                    name: Acme, Inc.
+                  display_name: Roger Rabbit
+                  member_id: dbmid:abcd1234
+  /2/sharing/mount_folder:
+    post:
+      tags:
+        - sharing
+      summary: mount_folder
+      description: >-
+        [mount_folder](https://www.dropbox.com/developers/documentation/http/documentation#sharing-mount_folder)
+
+
+        scope: `sharing.write`
+
+
+        The current user mounts the designated folder.
+
+        Mount a shared folder for a user after they have been added as a member.
+        Once mounted, the shared folder will appear in their Dropbox.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              example:
+                shared_folder_id: '84528192421'
+      parameters:
+        - name: Content-Type
+          in: header
+          schema:
+            type: string
+          exa
```

Comment:
Instead of repeating this content, shall we define this under the components requestBodies sections (component.requestBodies) and reference to operations?
```yml
components:
  requestBodies:
    CommonRequestBody:
      content:
          application/json:
            schema:
              type: object
              properties: {}
```
Reference to operation :

```
/2/users/get_space_usage:
    post:
   ....
      requestBody:
        $ref: "#/components/requestBodies/CommonRequestBody"
```
This will help to keep the OAS clear :)

Reply:
Thanks for the suggestion. I am not very familiar with OAS so missed that. BTW, used the ref name "EmptyRequestBody". Let me know if "CommonRequestBody" makes more sense, I will change that.

Reply:
@rahul1995, No worries at all 😃 !!,  I'm here to help you 🙌  , Yeah `EmptyRequestBody` also make sense +1 to have it.